cfme/configure/settings.py

```diff
@@ -7,8 +7,8 @@
 import cfme.web_ui.tabstrip as tabs
 import cfme.web_ui.toolbar as tb
 from cfme.web_ui import (AngularSelect, Form, Region, Select, fill, form_buttons, flash, Table,
-    ButtonGroup, Quadicon, CheckboxTree, Input, CFMECheckbox, BootstrapTreeview)
-from cfme.web_ui.menu import nav
+                         ButtonGroup, Quadicon, CheckboxTree, Input, CFMECheckbox,
```

Comment:
yes, personally i always end such lines with a comma and put the closing paren on the next line to create more visible grouping